app/workers/rent_mail_worker.rb

```diff
@@ -0,0 +1,12 @@
+class RentMailWorker
+  include Sidekiq::Worker
+
+  def perform(rent, action)
```

Comment:
Sidekiq doesn't accept complex objects as paramaters.

Also, consider using active job, is what rails provides and what we use currently at Flimper :registered: